### PR TITLE
Add Error handling for React Native SDK

### PR DESCRIPTION
**Pull request by cb-haripriyan**

### CODE REVIEW

File: android/src/main/java/com/chargebee/android/models/CBExceptionExtensions.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.chargebee.android.models
+
+import com.chargebee.android.exceptions.CBException
+import org.json.JSONObject
+
+fun CBException.payload(): String {
+  var payload = mapOf<String, Any?>(
+    "httpStatusCode" to this.httpStatusCode,
+    "message" to this.message,
+      "type" to this.type,
```
Comment (cb-sriramthiagarajan) [inline android/src/main/java/com/chargebee/android/models/CBExceptionExtensions.kt:10]: Is the formatting off? 🤔

File: src/ChargebeeError.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export interface ChargebeeError {
+  code: string;
+  message: string;
+  domain: string;
```
Comment (cb-sriramthiagarajan) [inline src/ChargebeeError.ts:4]: Why do we need `domain` and `userInfo` as part of the error object?

Comment (cb-haripriyan) [reply src/ChargebeeError.ts:4]: Not necessarily. But these are the common properties received at the JS layer from the native exception handlers.

File: android/src/main/java/com/chargebee/android/models/CBExceptionExtensions.kt (hunk 1 above)
Comment (cb-haripriyan) [reply android/src/main/java/com/chargebee/android/models/CBExceptionExtensions.kt:10]: Oh yeah. Let me correct this.

**Review by cb-sriramthiagarajan — APPROVED** [review]

**Pull request merged**